client/packages/common/src/utils/formatters/formatters.test.ts

```diff
@@ -48,12 +50,40 @@ describe('Formatter', () => {
   });
 
   it('naiveDateTime', () => {
-    expect(Formatter.toIsoString(null)).toBe(null);
-    expect(Formatter.toIsoString(new Date('1984/3/13'))).toBe(
-      '1984-03-12T12:00:00.000Z'
+    // While toIsoString is naive to the timezone, the timezone will still change.
+    // so when converting to GMT from the local timezone, the reference string will
+    // be wrong depending on the timezone.
+    // I've refactored to subtract timezone from naive string for. This will test both
+    // the naive interpretation and the timezone modification in the same test.
+
+    // eslint-disable-next-line new-cap
+    const timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+    const localalisedStartOfDay = new Date('1984/3/13');
+    // now subtract timezone which is daylight saving dependent
+    const utcStartOfDay = df.addMilliseconds(
+      localalisedStartOfDay,
+      dateFnsTz.getTimezoneOffset(timeZone, localalisedStartOfDay)
+    );
+    expect(Formatter.toIsoString(utcStartOfDay)).toBe(
+      '1984-03-13T00:00:00.000Z'
     );
+    expect(Formatter.toIsoString(null)).toBe(null);
+
+    const offsetInHours =
+      dateFnsTz.getTimezoneOffset(timeZone, new Date()) / 1000 / 60 / 60;
```

Comment:
Yes I'm wondering if that is the right place to put the test for getLocalDate, or if it needs a test at all. 

I am seeing that all of the other dateUtils functions don't have tests.

I wonder if I should just remove the toIsoString test (because this will fail depending on either what timezone the test is run OR if daylight saving time changes in nz). I think this would be good regardless because it would be confusing for international collaborators.

And/or create a new test file for dateUtils which could test the functions there.

Reply:
Have added a separate DateUtils.test.ts testing file